docs/src/main/sphinx/connector/delta-lake.rst

```diff
@@ -116,6 +116,12 @@ values. Typical usage does not require you to configure them.
       - Enables :ref:`Table statistics <delta-lake-table-statistics>` for
         performance improvements.
       - ``true``
+    * - ``delta.rename-managed-tables.enabled``
+      - Enable renaming of tables created without explicit location, this will
+        move the data on the underlying storage system. As moving data on the
```

Comment:
This needs to be clarified .. when does it move the data .. when you change the property value and restart Trino? Where does it move it to? 

i think this should be added asa separate section about renaming tables in the SQL support section of the connector docs and this property doc then just links to it.